consul/tests/test_unit.py

```diff
@@ -54,6 +54,25 @@ def test_get_nodes_with_service(aggregator):
     aggregator.assert_metric('consul.catalog.services_count', value=1, tags=expected_tags)
 
 
+def test_get_nodes_with_service_remove_service_tagkey(aggregator):
+    consul_check = ConsulCheck(common.CHECK_NAME, {}, [consul_mocks.MOCK_CONFIG])
+    consul_check.services_tags_include_service_name = False
+    consul_mocks.mock_check(consul_check, consul_mocks._get_consul_mocks())
+    consul_check.check(None)
```

Comment:
```suggestion
    no_tag_service_name_config = consul_mocks.MOCK_CONFIG
    no_tag_service_name_config['services_tags_include_service_name'] = False
    consul_check = ConsulCheck(common.CHECK_NAME, {}, [consul_mocks.MOCK_CONFIG])
    consul_mocks.mock_check(consul_check, consul_mocks._get_consul_mocks())
    consul_check.check(None)
```
Nit- can you set the config option here?